### DISCUSSION TITLE
Menus?

### DISCUSSION THREAD

**Discussion by rontarrant** [beeware/toga#2690] [General]
In all the docs on BeeWare and Toga, the only mention of menus I've found is how to get rid of the default menubar. This has led me to believe that menu systems aren't available... yet.

Are they planned?

Or if they already exist, where can I find the API/docs? Or even just an example.

**Comment by mhsmith** [comment]
The API is called Command, and it's implemented on all the major platforms except iOS. There's a search box at the top left of the documentation page which would allow you to find it:

<img width="1293" alt="Screenshot 2024-07-04 at 12 05 30" src="https://github.com/beeware/toga/assets/166954/c9124e28-87ba-4554-b726-192f8a5b86d1">

**Comment by rontarrant** [comment]
Thanks for the clarification. I did find that, but didn't put it all together. Fuzzy search has dulled me. I get it now.
